tests/test_helpers.py

```diff
@@ -323,3 +328,29 @@ def test_poorly_defined_functions(self):
         functions = [{'name': 'bogus', 'field': 'age'}]
         assert_raises(OperationalError, evaluate_functions, self.session,
                       self.Person, functions)
+
+
+class TestFieldChecksHandleTypeDecorators(DatabaseTestBase):
```

Comment:
I understand that much of the other code in this file doesn't follow the suggestion I'm about to make, but hear me out anyway :)

Unit tests should test behavior, not implementation. Functions in the `helpers` module are implementation details. I would rather see a unit test in `test_views.py`, testing whether a model with each of these fields (like the `Test` class you created below) can be used correctly with GET and POST requests, etc.